cli/src/athenapdf.js

```diff
@@ -118,8 +123,9 @@ if (process.platform === "linux") {
     };
 }
 
+// Toggle cache headers and add custom headers is specified
 const loadOpts = {
-    "extraHeaders": athena.cache ? "" : "pragma: no-cache\n"
+    "extraHeaders": (athena.cache ? "" : "pragma: no-cache\n") + athena.httpHeader.join("\n")
```

Comment:
I'm think if we should have a `extraHeaders` variable. Something like:

```
var extraHeaders = athena.httpHeader;

if (athena.cache) {
    extraHeaders += ["pragma: no-cache"];
}
```

Then we can change this line to:

```
"extraHeaders": extraHeaders.join("\n")
```

What do you think?